cmd/vela-cli/main.go

```diff
@@ -73,74 +73,74 @@ func main() {
 
 		// API Flags
 
-		altsrc.NewStringFlag(&cli.StringFlag{
+		&cli.StringFlag{
```

Comment:
We no longer need the `urfave/cli/v2/altsrc` package because we have a custom function to load our config.